templates/components/datatable.html.twig

```diff
@@ -225,7 +225,11 @@
                 {% if entries|length > 0 %}
                     {% for entry in entries %}
                         {% set row_massiveactions = entry['showmassiveactions']|default(showmassiveactions) %}
-                        <tr class="{{ loop.last ? "border-transparent" : "" }} {{ row_class|default('') }} {{ entry['row_class']|default('') }}"{% if row_massiveactions %} data-itemtype="{{ entry['itemtype'] }}" data-id="{{ entry['id'] }}"{% endif %}>
```

Comment:
The `data-itemtype` and `data-id` attributes were conditioned on the fact that massive action were enabled.
This break the itil cost form js as it need the id attribute despite not allowing massive actions.